.github/workflows/cicd.yaml

```diff
@@ -229,10 +229,6 @@ jobs:
         name: Setup k8s cluster
         with:
           k8s-version: v1.22
-      - name: Load Image
-        run: |
-          sudo k3s ctr images import ${GITHUB_SHA}_image.tar
-        shell: bash
```

Comment:
why is this not necessary anymore?

Reply:
because the `k8s-version-config` already loads the image if the `load-local-image` flag is set to `true`, which it is by default.

https://github.com/sse-secure-systems/connaisseur/blob/master/.github/actions/k8s-version-config/action.yaml#L30-L36

Reply:
when was that added?